app/controllers/items_controller.rb

```diff
@@ -236,6 +236,16 @@ def leave_waitlist
     end
   end
 
+  # PATCH
+  def toggle_status
+    @item.toggle_status
+
+    respond_to do |format|
```

Comment:
Save here and if saving fails return error message; Also make sure to check rights for toggling status.

Reply:
Thanks for this feedback. Please have a look at the new commit refactor: add save and permission check in toggle_status](https://github.com/hpi-swt2/bookkeeper-portal-red/pull/367/commits/8d8ba7853134fce505a85e27e46eb9d0956b2062) and let us know if there are still some issues